src/PrestaShopBundle/Form/Admin/AdvancedParameters/AuthorizationServer/ApiAccessType.php

```diff
@@ -93,6 +93,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ]),
                 ],
             ])
+            ->add('lifetime', TextType::class, [
```

Comment:
Isn't it possible to use an `IntegerType` instead? I believe in the mockup that's what is suggested since we can see the small arrows on the input